src/handlers/RotationGestureHandler.ts

```diff
@@ -0,0 +1,26 @@
+import createHandler from './createHandler';
+import {
+  BaseGestureHandlerProps,
+  baseGestureHandlerProps,
+} from './gestureHandlerCommon';
+
+export type RotationGestureHandlerEventPayload = {
+  rotation: number;
+  anchorX: number;
+  anchorY: number;
+  velocity: number;
+};
+
+export interface RotationGestureHandlerProps
+  extends BaseGestureHandlerProps<RotationGestureHandlerEventPayload> {}
+
+export type RotationGestureHandler = typeof RotationGestureHandler;
+// eslint-disable-next-line @typescript-eslint/no-redeclare -- backward compatibility; see description on the top of this file
```

Comment:
.

Reply:
Fixed in 4f58a3ff228fde7095fefb8cfec8843288fcb2af